module_utils/network_lsr/argument_validator.py

```diff
@@ -2461,6 +2463,14 @@ def _ipv6_is_not_configured(connection):
                     "if you need to use initscripts.",
                 )
 
+        # initscripts does not support ip.auto_dns, so raise errors when network
+        # provider is initscripts
```

Comment:
What about `PEERDNS`? https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/networking_guide/sec-configuring_ip_networking_with_ifcg_files

or is it that we just don't want to provide any new features for the initscripts provider?

Reply:
Yes, unless the user explicitly requested the new feature for the initscripts provider.